include/time.iol

```diff
@@ -93,11 +93,21 @@ type DateTimeType:void{
 	.second:int
 }
 
+
 type SetNextTimeOutRequest: int {
 	.operation?: string
 	.message?: undefined
 }
 
+type ScheduleTimeOutRequest: int {
+	.operation?: string
+	.message?: undefined
+	/*
+	*	Possible values are DAYS, HOURS, MICROSECONDS, MILLISECONDS, MINUTES, NANOSECONDS, SECONDS 
+	*/
+	.timeunit?: string
```

Comment:
Ah, I see now that it's in seconds, from the doc below. Please make it explicit here.

Isn't MILLISECONDS usually the default in this stuff?